src/libs/TransactionUtils.ts

```diff
@@ -45,16 +47,16 @@ function isDistanceRequest(transaction: OnyxEntry<Transaction>): boolean {
     return type === CONST.TRANSACTION.TYPE.CUSTOM_UNIT && customUnitName === CONST.CUSTOM_UNITS.NAME_DISTANCE;
 }
 
-function isScanRequest(transaction: Transaction): boolean {
+function isScanRequest(transaction: OnyxEntry<Transaction>): boolean {
     // This is used during the request creation flow before the transaction has been saved to the server
     if (lodashHas(transaction, 'iouRequestType')) {
```

Comment:
Could we have `lodashHas(transaction, 'iouRequestType')` replaced it with `transaction?.iouRequestType`?

Reply:
We can, but this was not done as this is not part of this issue.
Should we change it? wdyt?